pipeline-model-definition/src/main/resources/org/jenkinsci/plugins/pipeline/modeldefinition/properties/PreserveStashesJobProperty/help-buildCount.jelly

```diff
@@ -22,6 +23,11 @@
   ~ THE SOFTWARE.
   -->
 
-<div>
-    The number of most recent builds whose <code>stash</code>es should be preserved. Defaults to 1 if unspecified.
-</div>
\ No newline at end of file
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:f="/lib/form">
+    <j:getStatic var="maxCount" className="org.jenkinsci.plugins.pipeline.modeldefinition.properties.PreserveStashesJobProperty" field="MAX_SAVED_STASHES" />
```

Comment:
BTW it is generally easier to handle such things as simple instance properties (getters) on the `Descriptor`:

```xml
between 1 and ${descriptor.maxCount}
```